### PR TITLE
[TWEAK/FIX] Fixes hurting people you intend to help while performing surgery or repairing IPCs

### PR DESCRIPTION
**Pull request by Spaghetti-bit**
## What Does This PR Do

Addresses https://github.com/ParadiseSS13/Paradise/issues/25655

## Why It's Good For The Game

If I intend to help someone, I should help them, not maim them with a welder, brain, organ, circular saw, etc.

## Testing

Removed an IPC's arm, tried to apply welder. Got the funny "WHY WOULD YOU DO THAT?!" message.
Cycled intents, burn baby burn.
Also worked on organics. No more hurting people that are lying down and you're on help intent. 

## Changelog
:cl:
tweak: Help mode while intending to do a surgery or help a person on an operable surface now longer maims the person.
/:cl:

### CODE REVIEW

**Comment by iceberg112** [comment]
Wow!

**Review by Burzah — APPROVED** [review]

**Review by lewcc — CHANGES_REQUESTED** [review]
These feel like a bunch of arbitrary checks here. If a surgery is ongoing, there are already checks there which you can tie into, rather than having all of this handled on item attack and *guessing* if they're undergoing surgery.
Please see if you can't look into surgery code to at least make the surgery checks fit in there.

**Comment by Spaghetti-bit** [comment]
> These feel like a bunch of arbitrary checks here. If a surgery is ongoing, there are already checks there which you can tie into, rather than having all of this handled on item attack and _guessing_ if they're undergoing surgery. Please see if you can't look into surgery code to at least make the surgery checks fit in there.

This is intended for a catch all for help intent on tools used for the surgeries. If we're only checking DURING a surgery we'll have cases where someone is on the help intent and thwacks a person with a tool that can't start a surgery. Bone setter etc.

File: code/_onclick/item_attack.dm [hunk 1]
```diff
@@ -61,6 +61,10 @@
 	if(flags & (NOBLUDGEON))
 		return FALSE
 
+	if((is_surgery_tool_by_behavior(src) || is_organ(src) || tool_behaviour) && user.a_intent == INTENT_HELP && on_operable_surface(M) && M != user)
+		to_chat(user, "<span class='warning'>You lower the [name] and look at it, thinking to yourself... 'why would I hurt someone I'm trying to help?'</span>")
```
Comment (lewcc) [inline code/_onclick/item_attack.dm:65]:
```suggestion
		to_chat(user, "<span class='info'>You don't want to harm the person you're trying to help!</span>")
```

**Review by lewcc — COMMENTED** [review]
Just one thing, I think this message feels a little more consistent with general IC messages.

**Review by lewcc — APPROVED** [review]

**Pull request merged**